src/rendering/vcImageRenderer.h

```diff
@@ -57,4 +65,6 @@ udResult vcImageRenderer_Destroy();
 
 bool vcImageRenderer_Render(vcImageRenderInfo *pImageInfo, float encodedObjectId, const udDouble4x4 &viewProjectionMatrix, const udUInt2 &screenSize, double zScale, vcTexture *pTextureOverride = nullptr);
 
+const char *GetImageSaveFormatExtension(const vcImageSaveFormat &imageSaveFormat);
```

Comment:
Why not do something similar to `vcISToWorldSize` above? Also, you haven't scoped this function.